.circleci/continue_config.yml

```diff
@@ -60,7 +60,7 @@ commands:
 executors:
   build:
     docker:
-      - image: prestocpp/prestocpp-avx-centos:root-20230613
+      - image: public.ecr.aws/oss-presto/prestocpp:root-20240129
```

Comment:
Do we have some documentation somewhere that explains where these images come from and how to update them? Would it be possible to add a comment with a link here?

Reply:
We have instructions in the README here https://github.com/prestodb/presto/tree/master/presto-native-execution#build-using-dockerfile
The update will be handled by the Presto release management team. https://github.com/orgs/prestodb/teams/team-releases